packages/@orbit/store/src/push-payload.ts

```diff
@@ -0,0 +1,76 @@
+import { recordIdentityFrom } from './store-helpers';
+import Store from '@orbit/store';
+import { JSONAPISerializer } from '@orbit/jsonapi';
+import { KeyMap, Schema } from '@orbit/data';
+
+// NOTE: more may be added here
+export enum PAYLOAD_OPERATION {
+  ADD_RECORD = 'addRecord',
+  REPLACE_RECORD = 'replaceRecord',
+}
+
+// TODO: payload should be a valid `{ json:api }` payload
+export async function pushPayload(store: Store, payload: any, op = PAYLOAD_OPERATION.ADD_RECORD) {
```

Comment:
This is pulled from https://github.com/developertown/react-orbitjs/blob/master/src/utils/push-payload.ts

It's a bit of a hack, but we needed it for shoving data into the in-memory store from a non-`{ json:api }`-compliant endpoint (`/api/users/current-user`).

It also is used when receiving data from the websocket.